packages/ui-components/src/markdown/markdown.tsx

```diff
@@ -0,0 +1,293 @@
+import ReactMarkdown, { Options } from 'react-markdown';
+import remarkGfm from 'remark-gfm';
+
+import styled from '@darajs/styled-components';
+
+interface MarkdownProps extends Options {
+    /**
+     * The markdown string to render
+     */
+    markdown: string;
+}
+
+const CustomMarkdownWrapper = styled.div`
+    /*
+     * Apply some base styles to the markdown content so it looks good by default
+     * Inspired by https://tailwindcss.com/docs/typography-plugin
+    */
+    code {
+        font-size: 0.9rem;
+    }
+
+    p {
+        margin-top: 1.25rem;
+        margin-bottom: 1.25rem;
+        line-height: 1.5rem;
+
+        code {
+            padding: 0.1rem;
+            background-color: ${(props) => props.theme.colors.grey1};
+            border: 1px solid ${(props) => props.theme.colors.grey3};
+            border-radius: 0.25rem;
+        }
+    }
+
+    p,
+    tr {
+        math {
+            padding: 0;
+            font-size: 1.25rem;
+        }
+    }
+
+    /* stylelint-disable-next-line */
+    math {
+        padding: 0.5rem;
+        font-size: 1.5rem;
+    }
+
+    /* stylelint-disable-next-line */
+    highlight {
+        background-color: ${(props) => props.theme.colors.blue4};
+
+        &.highlight-focused {
+            color: ${(props) => props.theme.colors.background};
+            background-color: ${(props) => props.theme.colors.primary};
+        }
+    }
```

Comment:
This selector can probably be removed, I believe it was specific to studio so can be added there - doesn't need to be in the base

Reply:
good catch, removed